plan.go

```diff
@@ -2837,17 +2839,20 @@ func (r *emptyFieldsPlan) do(ctx *execCtx, f func(id interface{}, data []interfa
 
 type wrapFilterPlan struct {
 	*filterDefaultPlan
+	once  sync.Once
```

Comment:
can you expand more about what you think execCtx is good here.

Correct me if I'm wrong.

In case we have the guard on execCtx  , it meams we can pass execCtx to the select plan that is enclosed in the  `where exists (select...)`  and expect it to evaluate only once?